### PR TITLE
HW06 done

### PR DESCRIPTION
**Pull request by yestocode**

### CODE REVIEW

File: HW06_ex09_02.py [hunk 1]
```diff
@@ -13,11 +13,24 @@
 # Imports
 
 # Body
-
-
+def has_no_e():
```
Comment (danielsgriffin) [inline HW06_ex09_02.py]: goal was to have distinct functions so one, that opens the file, calls has_no_e.
we want to limit how much is performed by a function

but good job.

File: HW06_ex09_04.py [hunk 2]
```diff
@@ -18,10 +18,34 @@
 
 # Body
 
+def uses_only(word,letters):
+	count=0
```
Comment (danielsgriffin) [inline HW06_ex09_04.py]: we can probably do this w/o flags

File: HW06_ex09_04.py [hunk 3]
```diff
@@ -18,10 +18,34 @@
...
+
+
+	if flag==True:
+		return True
+
 
 ##############################################################################
 def main():
-    pass  # Call your function(s) here.
+	fo=open("words.txt","r")
```
Comment (danielsgriffin) [inline HW06_ex09_04.py]: Good job.

minor point: look at the spacing around your operators and https://www.python.org/dev/peps/pep-0008/#other-recommendations

File: HW06_ex09_05.py [hunk 4]
```diff
@@ -16,11 +16,33 @@
 # Imports
 
 # Body
+def uses_all(word,letters):
+
```
Comment (danielsgriffin) [inline HW06_ex09_05.py]: same as above, we don't need flags.

File: HW06_ex09_06.py [hunk 5]
```diff
@@ -13,11 +13,24 @@
 # Imports
 
 # Body
-
+def is_abecedarian(word):
```
Comment (danielsgriffin) [inline HW06_ex09_06.py]: good

glance at Richa's for an alt. example

**Comment by hatsy** [comment]
Could you please remove the file with name of participants. 
Thanks.

**Pull request closed without merging**